### ISSUE TITLE
Bot getting kicked for outdated client?

### ISSUE THREAD

**Issue by safazi** [PrismarineJS/mineflayer#443]
My bot is getting kicked from my 1.10.2 server

Here's my error.

> I got kicked for {"text":"Outdated client! Please use 1.10.2"}

Is minecraft-protocol outdated? Is there some way I can manually change the version sent to the server?

**Comment by roblabla** [comment]
You can try to specify the version in the constructor :

```js
var mc = require('minecraft-protocol');
mc.createClient({
  version: '1.10',
  // ... Other options
});
```

Though I think it's supposed to find the version automatically.

**Comment by rom1504** [comment]
Version:false for auto detection.
But that's for node-minecraft-protocol
Mineflayer doesn't quite support anything but 1.8 yet.
Feel free to contribute in supporting 1.10 though !

**Comment by growlf** [comment]
Hi Rom1504!  What areas need help to get MF to 1.10?  Is there a list we can poke at and see what can be done?  Or is it a chaos cloud that we cant really attack in small slices?  I see that MCP is already supporting 1.10, so I suspect you have a clear idea what needs doing already.

**Comment by imnofox** [comment]
@growlf The biggest stepping stone is to get the chunk parser updated to support 1.9+ (#417)

**Comment by rom1504** [comment]
closed in favor of #449
